app/cdap/components/Replicator/Create/Content/SelectColumnsWithTransforms/TransformAdd.tsx

```diff
@@ -135,10 +135,12 @@ export default function TransformAddButton({
           dense: true,
         }}
       >
-        <MenuItem onClick={(event) => {
-          handleMenuClick('rename');
-          setSubMenuAnchorEl(event.currentTarget);
-        }}>
+        <MenuItem
+          onClick={(event) => {
```

Comment:
Can you remove these changes? I don't think they're relevant to this PR.

Reply:
you are right , these are not related to current PR. But indentation being corrected automatically and got added to PR.